cmd/dao_tests/pubkey_resource_test.go

```diff
@@ -0,0 +1,137 @@
+//go:build integration
+// +build integration
+
+package main
+
+import (
+	"context"
+	"github.com/RHEnVision/provisioning-backend/internal/testing/factories"
+	"testing"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	_ "github.com/RHEnVision/provisioning-backend/internal/dao/sqlx"
```

Comment:
Should we move this one into `main_test`? Just a thought. Not important.

Reply:
Yeah all side-effect imports should be in the `main.go` so you don't have to think about it.

Reply:
Great. I will have just removed this everywhere and moved it to main.go.